rpp30-boc-meltdown/biteshare
# Setup navigation

## Description
This PR contains the navigation setup of our **biteshare** app. I have created 4 different tabs for our app according to the wireframe. This PR also includes the necessary file structure. Please write your code in the respective component as it depends on the navigation to work. I have also included the headers for each of the screens. If you are working on the Explore page, please use `src/features/ExploreView/Explore.Screen.js` file. Similarly, for anything for the Join Screen, please use `src/features/JoinView/Join.Screen.js`, for CurrentSession,  use `src/features/CurrentSessionView/CurrentSession.Screen.js`, for Profile, use 'src/features/ProfileView/Profile.Screen.js'
For any general component, not specific to any particular feature, use the `components` folder under `src` to create your component. For example, I have created two components `<SafeArea />` and `<BackButton />` which can be used anywhere in the codebase. For any other components who are responsible for any specific feature/screen, create them in either `ExploreView`, `JoinView`, `CurrentSessionView`, or `ProfileView` folders.

## How to test
To test this PR, pull the `setup-navigation` branch into your local machine by first running `git fetch` and then `git checkout setup-navigation` . You should see the following images. Also, the navigation system is functioning. So click the tabs and verify if it redirects you to the respective screen.  
![Screen Shot 2021-12-22 at 8 08 43 PM](https://user-images.githubusercontent.com/80971293/147172464-a26b98e0-607e-4a6e-b03f-36902a167d32.png)
![Screen Shot 2021-12-22 at 8 09 14 PM](https://user-images.githubusercontent.com/80971293/147172498-6509245a-6b29-45cd-91d8-2328e0c30526.png)
![Screen Shot 2021-12-22 at 8 09 35 PM](https://user-images.githubusercontent.com/80971293/147172526-3768fa56-5ecd-4138-9e9b-12997637384d.png)
![Screen Shot 2021-12-22 at 8 09 57 PM](https://user-images.githubusercontent.com/80971293/147172550-3c8ec3bd-aa76-401a-99f1-d0695c56cf2b.png)


## Notes
You might have a merge conflict at this point as I commented out the <DummyComponent /> and some other test code. Please resolve accordingly.

## Issue tracking
https://trello.com/c/TqbnEYaB
